kivy/lang/builder.py

```diff
@@ -296,7 +296,9 @@ def load_file(self, filename, encoding='utf8', **kwargs):
 
             `encoding`: File charcter encoding. Defaults to utf-8,
         '''
-        filename = resource_find(filename) or filename
+
+        # Cache here can give problems since Clock sometimes does not exist.
```

Comment:
Why is the presence of Clock an issue? It doesn't seem to be used in `resource_find`.

Reply:
Cache uses Clock

Reply:
Are you worried that the timeout won't happen because the clock is not running? Because the clock should be available whether it is running or not so this shouldn't cause issues beyond the timeout not happening.

Reply:
https://github.com/kivy/kivy/runs/1513086942

This actually happens in building docs, very strange. I've reverted it so you can investigate, I am unable to reproduce it on windows.
```
'lastaccess': Clock.get_time(),
AttributeError: 'NoneType' object has no attribute 'get_time'
```

Reply:
It's because of this: https://github.com/kivy/kivy/blob/master/kivy/clock.py#L1155. The `KIVY_DOC_INCLUDE` is required to be able to generate rtd docs because we can't create a window there.

Long term the solution is to remove `KIVY_DOC_INCLUDE` and make kivy imports less aggressive so running on rtd will work. But for now, I guess for now we can do the hacky thing and disable cache in this environment as in my suggested edits.

Reply:
I see. I've had my own tests fail on occasion as well, usually due to sp() or something making a window